src/qt/bitcoingui.cpp

```diff
@@ -412,6 +419,25 @@ void BitcoinGUI::createActions()
                 action->setEnabled(false);
             }
         });
+        connect(m_restore_wallet_action, &QAction::triggered, [this] {
+            QString backup_file = GUIUtil::getOpenFileName(this,
+                                                          //: The title for Restore Wallet File Windows
+                                                          tr("Load Wallet Backup"), QString(),
+                                                          //: The file extension for Restore Wallet File Windows
+                                                          tr("Wallet Data File (*.dat)"), nullptr);
+            if (backup_file.isEmpty()) return;
+
+            bool walletNameOk;
+            //: Title of the Restore Wallet input dialog (where the wallet name is entered)
+            QString walletName = QInputDialog::getText(this, tr("Restore Name"), tr("Wallet Name:"), QLineEdit::Normal, "", &walletNameOk);
```

Comment:
Please follow our naming [convention](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c):
```suggestion
            QString wallet_name = QInputDialog::getText(this, tr("Restore Name"), tr("Wallet Name:"), QLineEdit::Normal, "", &walletNameOk);
```

Reply:
Done in https://github.com/bitcoin-core/gui/commit/e7a3f698b5f3f7dde8632c4911abd4e5bc1f06cb.